_includes/components/choice-menu-items.md

```diff
@@ -0,0 +1,28 @@
+#### Choice menu items
+
+Choice menu items are available as Split toolbar button menu items. For a description of how choice menu items are, see: [Split toolbar buttons]({{site.baseurl}}/ui-components/typesoftoolbarbuttons/#splitbutton).
```

Comment:
Should the capital `S` be in split here, as that feels wrong?

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1573/commits/d6f9cf4a8078de019a983938369aca2dfef00e0b